src/phpDocumentor/Reflection/Php/Factory/File/CreateCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\Php\Factory\File;
+
+
+use phpDocumentor\Reflection\Php\StrategyContainer;
+
+/**
+ * File Create command is used by the File Factory Strategy.
+ * The command is passed to the registered middle ware classes.
+ */
+final class CreateCommand
+{
+    /**
+     * @var Adapter
+     */
+    private $adapter;
+
+    /**
+     * @var string
+     */
+    private $object;
+
+    /**
+     * @var StrategyContainer
+     */
+    private $strategies;
+
+    /**
+     * Initializes this command.
+     *
+     * @param Adapter $adapter
+     * @param string $object
+     * @param StrategyContainer $strategies
+     */
+    public function __construct(Adapter $adapter, $object, StrategyContainer $strategies)
+    {
+
+        $this->adapter = $adapter;
+        $this->object = $object;
+        $this->strategies = $strategies;
+    }
+
+    /**
+     * Returns the adapter used by the command handler for this command.
+     *
+     * @return Adapter
+     */
+    public function getAdapter()
+    {
+        return $this->adapter;
+    }
+
+    /**
+     * Returns the path of the processed file.
+     *
+     * @return string
+     */
+    public function getObject()
```

Comment:
when I read the name getObject I got confused because I was expecting a real (albeit generic) object to be returned; it was only in the DocBlock that I encountered that this is the path of the file that is going to be processed. Perhaps a more specific name is needed or an explanation in the DocBlock why it is called getObject?